src/game/GameStateMatch.cpp

```diff
@@ -28,25 +29,15 @@ bool GameStateMatch::load() {
     GameManager::instance()->setBoundary(0, 0, MAP_WIDTH, MAP_HEIGHT);
 
 
-    //create Weapon for weapon Drop only till sotre is implemented****************
-    Rifle w(GameManager::instance()->generateID());
-    ShotGun w2(GameManager::instance()->generateID());
-    GameManager::instance()->addWeapon(std::dynamic_pointer_cast<Weapon>(std::make_shared<Rifle>(w)));
-    GameManager::instance()->addWeapon(std::dynamic_pointer_cast<Weapon>(std::make_shared<ShotGun>(w2)));
     // Create Dummy Entitys
     GameManager::instance()->createMarine(1000, 500);
-    GameManager::instance()->createWeaponDrop(1200, 500, w.getID());
-    GameManager::instance()->createWeaponDrop(1200, 300, w2.getID());
 
 
     //createStores
-    GameManager::instance()->createWeaponStore(500, 500);
+    GameManager::instance()->createWeaponStore(storeX, storeY);
 
     //createDropPoint
-    GameManager::instance()->createDropPoint(100 ,100);
-    GameManager::instance()->createDropPoint(100, 300);
-    GameManager::instance()->createDropPoint(300 ,100);
-    GameManager::instance()->createDropPoint(300, 300);
+    GameManager::instance()->createDropZone(dropZoneX , dropZoneY, dropZoneSize);
```

Comment:
this is not a solution use `static constexpr int` these are constants not variables!